src/services/transpile.ts

```diff
@@ -57,6 +57,8 @@ namespace ts {
         options.rootDirs = undefined;
         options.declaration = undefined;
         options.composite = undefined;
+        options.incremental = undefined;

```

Comment:
Probably not for 3.6.3, but should this sort of thing be declared in `commandLineParser.ts` so that every option has to declare it up front?

Reply:
good idea and will change that but we should do this for 3.6.3 since it is a regression after fixing a bug where we were not emitting tsbuildinfo file if there was no information to emit which was the case in transpile mode and hence worked correctly in 3.5 but after that bug was fixed the tsbuildinfo file is emitted if this fix is not taken and transpile only expects one file to be written.

Reply:
Oh, I meant I agreed that this should go into 3.6.3, but I didn't want you to go through the effort of updating `commandLineParser.ts`. Thanks for doing it though, great work!